Resync web-platform-tests/permissions from upstream

marcoscaceres opened the pull request:
#### 122ad287d93418e4992710cdfaf882446b5f5349
<pre>
Resync web-platform-tests/permissions from upstream
<a href="https://bugs.webkit.org/show_bug.cgi?id=247406">https://bugs.webkit.org/show_bug.cgi?id=247406</a>
rdar://101891698

Reviewed by Tim Nguyen.

Tests updated to 1b73dcd.

* LayoutTests/imported/w3c/web-platform-tests/permissions/all-permissions-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/permissions/non-fully-active.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/permissions/non-fully-active.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/permissions/permissions-garbage-collect.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/permissions/permissions-garbage-collect.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/permissions/resources/empty.html: Added.

Canonical link: <a href="https://commits.webkit.org/256557@main">https://commits.webkit.org/256557@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/f702142ff891adff6d006963b4698cef9c2954e1

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/95983 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/5232 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/29024 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/105542 "Built successfully") | [✅ 🛠 🧪 win](https://ews-build.webkit.org/#/builders/10/builds/165866 "Built successfully and passed tests") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/5340 "Built successfully") | [✅ 🛠 mac-debug](https://ews-build.webkit.org/#/builders/71/builds/33994 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/88365 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/101365 "Built successfully") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/101644 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/3942 "Passed tests") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/82588 "Built successfully") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/30980 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/85791 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/87698 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/73813 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/66/builds/39725 "Built successfully") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/19226 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/67/builds/37403 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/20567 "Passed tests") | | 
| [❌ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/4535 "Failed to push commit to Webkit repository") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/63/builds/42000 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/60/builds/43180 "Passed tests") | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/65/builds/43886 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/39827 "Passed tests") | | 
<!--EWS-Status-Bubble-End-->

nt1m reviewed (approved):
Looks fine if EWS is green, but please include upstream commit hash/link in the commit message.

marcoscaceres committed:
Resync web-platform-tests/permissions from upstream
https://bugs.webkit.org/show_bug.cgi?id=247406
rdar://101891698

Reviewed by Tim Nguyen.

Tests updated to 1b73dcd.

* LayoutTests/imported/w3c/web-platform-tests/permissions/all-permissions-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/permissions/non-fully-active.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/permissions/non-fully-active.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/permissions/permissions-garbage-collect.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/permissions/permissions-garbage-collect.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/permissions/resources/empty.html: Added.

Canonical link: https://commits.webkit.org/256557@main
